src/solver/misc/include/antares/solver/misc/options.h

```diff
@@ -48,8 +48,11 @@ class Settings
     int ignoreWarningsErrors = 0;
     //! Ignore constraints
     bool ignoreConstraints = false;
-    //!
+
+    //! Run the TS generator only
     bool tsGeneratorsOnly = false;
+    std::string clustersToGen = "";
```

Comment:
It would be best to have the vector of pairs here (actually better vector of small structs), and have the parsing done at options parsing time.
This way:
1. we will catch errors early
2. the business logic will not be cluttered with parsing code

Reply:
The parsing should be done in the separated tool
We need to keep it here for the moment to support the in memory mode